entrypoint.sh

```diff
@@ -1,187 +1,21 @@
-  --wasbs storageAccountName containerName sasKeyMode \ Enable WASBS and configure its params.
-          sasKeyName(or storageAccountKeyName)          If sasKeyMode=true - provide sasKeyName as last arg,
-                                                        else provide storageAccountKeyName as last arg
-  --events-dir events-dir                               Set events dir
-  -h | --help                                           Prints this message.
-EOF
-}
-
-function parse_args {
-  while [[ $# -gt 0 ]]
-  do
-    case "$1" in
-      --pvc)
-        enablePVC=true
-        shift
-        continue
-      ;;
-      --gcs)
-        enableGCS=true
-        if [[ -n "$2" ]]; then
-          gcloudKey=$2
-          shift 2
-          continue
-        else
-          printf '"--gcs" requires a non-empty option argument.\n'
-          usage
-          exit 1
-        fi
-      ;;
-      --s3)
-      if [[ -n "$4" ]]; then
-        enableS3=true
-        enableIAM=$2
-        accessKeyName=$3
-        secretKeyName=$4
-        shift 4
-        continue
-      else
-        printf '"--s3" require three non-empty option arguments.\n'
-        usage
-        exit 1
-      fi
-      ;;
-      --wasbs)
-      if [[ -n "$5" ]]; then
-        enableWASBS=true
-        storageAccountName=$2
-        containerName=$3
-        sasKeyMode=$4
-        if [ "$sasKeyMode" == "true" ]; then
-          sasKeyName=$5
-        else
-          storageAccountKeyName=$5
-        fi
-        shift 5
-        continue
-      else
-        printf '"--wasbs" require four non-empty option arguments.\n'
-        usage
-        exit 1
-      fi
-      ;;
-      --events-dir)
-        if [[ -n "$2" ]]; then
-          eventsDir=$2
-          shift 2
-          continue
-        else
-          printf '"--events-dir" requires a non-empty option argument.\n'
-          usage
-          exit 1
-        fi
-      ;;
-      -h|--help)
-        usage
-        exit 0
-      ;;
-      --)
-        shift
-        break
-      ;;
-      '')
-        break
-      ;;
-      *)
-        printf "Unrecognized option: $1\n"
-        exit 1
-      ;;
-    esac
-    shift
-  done
-}
-
-parse_args "$@"
+if [[ "${#}" -lt 2 ]]; then
+  printf 'Usage: %s eventsDirPath serverSideEncryptionAlgorithm\n' "${BASH_SOURCE[0]##*/}" >&2
+  exit 1
+fi
 
-# Check whether there is a passwd entry for the container UID
-uid=$(id -u)
-gid=$(id -g)
-# turn off -e for getent because it will return error code in anonymous uid case
-set +e
-uid_entry=$(getent passwd ${uid})
-set -e
+declare SPARK_HISTORY_OPTS=""
 
-# If there is no passwd entry for the container UID, attempt to create one
-if [[ -z "${uid_entry}" ]] ; then
-    if [[ -w /etc/passwd ]] ; then
-        echo "$uid:x:$uid:$gid:anonymous uid:${SPARK_HOME}:/bin/false" >> /etc/passwd
-    else
-        echo "Container entrypoint.sh failed to add passwd entry for anonymous UID"
-    fi
-fi
+readonly eventsDir="${1}"
+readonly serverSideEncryption="${2}"
```

Comment:
Please use uppercase and snake case for constants.
I would also stay close to the name of the spark property:

```suggestion
readonly SPARK_LOG_DIR="${1}"
readonly S3_SSE_ALGORITHM="${2}"
```

Reply:
Sounds good, I will go with your suggestion.